### ISSUE TITLE
Improve the read/write speed of it

### ISSUE THREAD

**Issue by Zero3K** [Zero3K/ERAM#12]
It would be nice if that were to be done (if possible).

**Comment by Zero3K** [comment]
I don't think it can be made any faster. I did notice a slight increase in speed when switching from a Debug build to a Release build though.

**Comment by Zero3K20** [comment]
I have made some new releases on my fork of it at https://github.com/Zero3K20/ERAM. I am currently unable to login into my Zero3K account because of 2FA.
